clap-v3-utils/src/input_validators.rs

```diff
@@ -59,6 +59,10 @@ where
 }
 
 // Return an error if a pubkey cannot be parsed.
+#[deprecated(
+    since = "1.17.2",
```

Comment:
this pr is targeting master, why are we deprecating it in an already tagged/released beta version? (throughout)

Reply:
Oops, my mistake. I changed this to `1.18.0`.

Reply:
I guess it just depends on whether we plan to backport this to v1.17. If so, it makes sense for this to be the beta version. But yeah, should be v1.17.4 at this point.